examples/more/external-project-add/.gitignore

```diff
@@ -0,0 +1,2 @@
+3rdparty/
+build/
```

Comment:
We ignore `build/` folders in the root `.gitignore` so this can be omitted

Reply:
> 
> 
> Hey @smeualex,
> 
> Thank you for suggesting the change. I don't think it can be accepted in its current form but I've left some suggestions for simplifications/improvements.
> 
> My general feedback is to try and pair things back even more and remove anything that feels redundant,
> 
> Let me know what you think,
> 
> Thanks again,
> 
> Tom

Hey Tom,

To be honest, this is way more feedback than I expected; always happy to learn to do things better - not just to make them work :).

Thank you very much for the comments and I think that you can abandon this PR  (or leave it pending untill I'll have a bit more time to change it).

Alex.

Reply:
Hey Alex,

Lol I'm sorry 🙈  I really like the general premise it's just getting it super polished and focussed would be amazing.

That's quite alright! Thank you for your good-natured reply! 🙂 It's absolutely fine to leave pending, I'm sorry I didn't have time this evening to go through your other comments but I'll make sure to respond at the weekend 👍 

Thanks!


Tom